dask/bytes/core.py

```diff
@@ -0,0 +1,164 @@
+import io
+import sys
+
+from toolz import merge
+
+from .compression import seekable_files, files as compress_files
+from .utils import SeekableFile
+from ..delayed import delayed
+from ..utils import system_encoding
+
+delayed = delayed(pure=True)
+
+_read_bytes = dict()
+_open_files = dict()
+_open_text_files = dict()
```

Comment:
A comment on each of these lookups on what they contain might be nice. It's not immediately clear without some digging.